### PR TITLE
Down-grading hazelcast version for spring-data-hazelcast example

### PR DESCRIPTION
**Pull request by dineshbhagat**
Ref: https://github.com/hazelcast/spring-data-hazelcast/issues/14

### CODE REVIEW

File: hazelcast-integration/spring-data-hazelcast-chemistry-sample/pom.xml [hunk 1]
```diff
@@ -22,7 +22,9 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <java.version>1.8</java.version>
-        <spring-data-hazelcast.version>2.1</spring-data-hazelcast.version>
+        <!-- adding specific Hazelcast version, since spring-data-hazelcast do not support Hazelcast 4.1-SNAPSHOT version -->     
+        <hazelcast.version>3.11.7</hazelcast.version>
```
Comment (mesutcelik) [inline hazelcast-integration/spring-data-hazelcast-chemistry-sample/pom.xml:26]: is there a reason not to use 3.12.7?

Comment (dineshbhagat) [reply hazelcast-integration/spring-data-hazelcast-chemistry-sample/pom.xml:26]: The major version I could find [here](https://github.com/hazelcast/hazelcast/releases) is 3.11.7, another version is [3.12.6-adobe](v3.12.6-adobe )

Comment (mesutcelik) [reply hazelcast-integration/spring-data-hazelcast-chemistry-sample/pom.xml:26]: You can see that 3.12.7 is latest one from download link for 3.12.z release line.
https://hazelcast.org/imdg/download/

@Holmistr @hasancelik Do you know why 3.12.7 is not listed as release but a tag on releases page?
https://github.com/hazelcast/hazelcast/releases

Comment (dineshbhagat) [reply hazelcast-integration/spring-data-hazelcast-chemistry-sample/pom.xml]: verified against `3.12.7` version too. works fine.

Comment (Holmistr) [reply hazelcast-integration/spring-data-hazelcast-chemistry-sample/pom.xml]: @mesutcelik No idea :-/

**Pull request merged**